API/Services/Implementations/MatchesService.cs

```diff
@@ -74,4 +95,16 @@ public async Task<bool> ExistsAsync(int id) =>
 
     public async Task DeleteAsync(int id) =>
         await matchesRepository.DeleteAsync(id);
+
+    private async Task<ICollection<PlayerCompactDTO>> GetPlayerCompactsAsync(MatchDTO match)
+    {
+        IEnumerable<int> playerIds = match.Games
+            .Select(x => x.Scores.Select(y => y.PlayerId))
+            .SelectMany(x => x)
```

Comment:
```suggestion
            .SelectMany(g => g.Scores.Select(s => s.PlayerId))
```
Think this can be condensed, also lambda var names